style.css

```diff
@@ -0,0 +1,241 @@
+  width: 100%;
+  position: fixed;
+  top: 0px;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: flex-end;
+  padding-bottom: 20px;
+}
+
+#title {
+  font-size: 55px;
+  font-family: 'Raleway', sans-serif;
+}
+
+#sub-title {
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-size: 30px;
+}
+
+#headings-section {
+  margin-top: 220px;
+  background-color: rgba(246, 299, 215, 0.3);
+}
+
+#body-copy {
+  background-color: rgba(246, 299, 215, 0.3);
+}
+
+#address {
+  background-color: rgba(246, 299, 215, 0.3);
+}
+
+small, .small {
+  color: #6d7370;
+  font-size: 0.8em;
+}
+
+#blockquotes {
+  background-color: rgba(246, 299, 215, 0.3);
+}
+
+#buttons {
+  background-color: rgba(246, 299, 215, 0.3);
+}
+
+p {
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-size: 20px;
+  line-height: 1.5;
+  margin-bottom: 10px;
+}
+
+code {
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+address {
+  font-size: 20px;
+  line-height: 1.5;
+  margin-bottom: 7px;
+  font-style: normal;
+  font-family: 'Open Sans Condensed', sans-serif;
+}
+
+blockquote {
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-size: 75%;
+  margin: 0 0 20px;
+  padding: 10px 20px;
+  border-left: 7px solid #f78671;
+}
+
+.flex-container.sample-text {
+  display: flex;
+  flex-direction: column;
+  background-color: rgba(201, 210, 203, 0.8);
+  margin-bottom: -20px;
+  padding: 5px 200px;
+}
+
+.flex-container.sample-text.buttons {
+  display: flex;
+  flex-direction: row;
+}
+
+.section-title {
+  font-family: 'Raleway', sans-serif;
+  color: #2f3733;
+  background-color: #86baaa;
+  text-align: center;
+  margin-top: 0px;
+  border-radius: 5px;
+}
+
+.element-color {
+  color: #6b9488;
+}
+
+.element-in-text {
+  color: #6b9488;
+  background-color: #d9dfda;
+}
+
+.class-name {
+  color: #f78671;
+}
+
+.class {
+  color: #c9d2cb;
+}
+
+
+.tab {
+  margin-left: 40px;
+}
+
+h1, .vast-h1 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 2em;
+}
+
+h2, .vast-h2 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1.5em;
+}
+
+h3, .vast-h3 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1.17em;
+}
+
+h4, .vast-h4 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: 1em;
+}
+
+h5, .vast-h5 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: .83em;
+}
+
+h6, .vast-h6 {
+  color: #2f3733;
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-weight: 700;
+  font-size: .67em;
+}
+
+.vast-lead {
+  font-family: 'Open Sans Condensed', sans-serif;
+  font-size: 27px;
+  color: #828784;
+}
+
+.sample-code {
+  background-color: rgba(47, 55, 51, 0.8);
+  color: #f6e5d7;
+  padding: 0px 200px 5px 200px;
+}
+
+button.vast-btn, input.vast-btn {
```

Comment:
``` border: none; ```

i believe will take away the depth and grey border on the button. bootstrap is famous for its flat buttons.